modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/submodel/executor/create/steps/impl/RecordProcessUtils.java

```diff
@@ -57,12 +61,10 @@ private boolean isDateFormatField(JsonObject jObject) {
 		return jObject.get("format") != null && "date-time".equals(jObject.get("format").getAsString());
 	}
 
-	private boolean isNumberTypeField(JsonObject jObject, String fieldValue) {
-
-		if (fieldValue != null && !fieldValue.isBlank() && jObject.get("type") != null
-				&& jObject.get("type").isJsonArray()) {
+	private boolean isNumberTypeField(JsonObject jObject) {
+		JsonElement jsonElement = JsonParser.parseString("number");
```

Comment:
Optional:: jsonElement is effectively final, consider to move it out of function and make final

Reply:
no other suggestions for the code, LGTM.

Reply:
@dvasunin thanks for quick help, if your fine with it could you add your "LGTM" to this PR

Reply:
LGTM

Reply:
i mean as content review 
![image](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/assets/121097161/02b97b1d-ec37-41fe-ae3b-9713de20929a)